python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/aaa_authentication.schema.yml

```diff
@@ -20,6 +20,14 @@ keys:
               - "group tacacs+ local"
               - "group MYGROUP none"
               - "group radius group MYGROUP local"
+          command_api:
+            type: str
+            description: |
```

Comment:
```suggestion
            type: str
            # TODO: Revisit to see if the feature is now visible in EOS and update description
            description: |
```

Reply:
@gmuloc do you want me to add this comment ?

Reply:
let me just commit this :) sorry I was distracted!